scg-system/src/main/java/io/telicent/backup/services/DatasetBackupService.java

```diff
@@ -756,4 +759,40 @@ private void compressAndStoreBackupMetadata(ObjectNode response, String dirPath)
         writeObjectNodeToFile(response, dirPath + JSON_INFO_SUFFIX);
     }
 
+    /**
+     * A utility method that takes the size of something in bytes and represents it in
+     * a more easy to grasp format.
+     * @param bytes size of file in question
+     * @return a string representation in KB, MB, GB, TB or PB
+     */
+    public static String humanReadableByteCount(long bytes) {
```

Comment:
There's a `FileUtils.byteCountToDisplaySize()` static method that we already use in the initial compaction module that already does this for us